ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml

```
@@ -0,0 +1,175 @@
+base_images:
+  builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  to: multiarch-manager-operator
+- dockerfile_literal: |
+    FROM quay.io/operator-framework/operator-sdk:v1.26.0 as osdk
+    FROM src
+    COPY . /code
+    COPY --from=osdk /usr/local/bin/operator-sdk /usr/local/bin/
+    RUN chmod -R g+rwX /code
+    WORKDIR /code
+  from: src
+  inputs:
+    builder:
+      as:
+      - golang:1.21
+  to: operator-sdk-src
+operator:
+  bundles:
+  - as: multiarch-manager-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: controller:latest
+    with: pipeline:multiarch-manager-operator
+promotion:
+  to:
+  - excluded_images:
+    - operator-sdk-src
+    namespace: origin
+    tag: release-4.16
+releases:
+  initial:
+    integration:
+      name: "4.15"
```

```suggestion
      name: "4.16"
```

also, would we start pivoting to 4.16? We'd need this done for the main branch too.

---

right...4.16 is better unless we plan on creating a release-4.15